### PR TITLE
New module: BACPHLIP

### PR DESCRIPTION
**Pull request by CarsonJM**
## PR checklist

Closes #XXX 

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/bacphlip/main.nf [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    bacphlip \\
+        -i $fasta \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        bacphlip: 0.9.6
```
Comment (sateeshperi) [inline modules/nf-core/bacphlip/main.nf:31]: version info not provided by CLI?

Comment (CarsonJM) [reply modules/nf-core/bacphlip/main.nf:31]: Correct, no version info provided

Comment (CarsonJM) [reply modules/nf-core/bacphlip/main.nf]: Thanks for the information on how to deal with a scenario like this

**Review by sateeshperi — APPROVED** [review]
gr8 job!

**Comment by CarsonJM** [comment]
Thanks for the help once again @sateeshperi ! I am excited to help nf-core have access to more phage-related modules!

**Pull request merged**